docs/api/contrib/missing/bar.rst

```diff
@@ -1,7 +1,7 @@
 .. -*- mode: rst -*-
 
 MissingValues Bar
-=============================
+=================
```

Comment:
Updated headings format to match Sphinx specifications.

Reply:
@Kautumn06 thank you very much for fixing this, although @ndanielsen just fixed this as well in #668 -- no need to do anything, but I wanted you both to be aware in case there was a merge conflict when we got it sorted.